2021/34xxx/CVE-2021-34604.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
('Invalid control character at: line 6 column 27 (char 122)',)

Reply:
fixed in 2nd commit